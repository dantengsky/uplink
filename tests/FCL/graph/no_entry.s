/* This test has no initial state, but the initial state is mandatory */

transition a -> terminal;

@a
end() {
  terminate("bye");
}
