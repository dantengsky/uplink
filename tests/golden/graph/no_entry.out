We have the following graph errors:
Cannot reach state 'a' from initial state in graph.
Cannot reach state 'terminal' from initial state in graph.